### PR TITLE
Extend atlas names

### PR DESCRIPTION
**Pull request by arschat**
### Release notes

For `hca_bionetwork.json` schema:
- Extend the controlled vocabulary of atlas names in `hca_tissue_atlas`
#1583 

### Reviews requested

- Need 5 Reviewers to approve because this is a patch update

### CODE REVIEW

File: json_schema/update_log.csv [hunk 1]
```diff
@@ -1 +1,2 @@
 Schema,Change type,Change message,Version,Date
+module/project/hca_bionetwork,patch,Extend atlas names in hca_bionetwork,,
```
Comment (ESapenaVentura) [inline json_schema/update_log.csv:2]: We should revisit what minor/patch versions are like. Thanks for following the guide https://github.com/HumanCellAtlas/metadata-schema/blob/master/docs/evolution.md#schema-versioning but I think extending an enumeration should be considered a minor patch, as in my opinion any change in the schema that involves change in the validation rules should be at least a minor change. Anyways, this is good to go as per our specs! 🥇

**Review by ESapenaVentura — APPROVED** [review]

File: json_schema/module/project/hca_bionetwork.json [hunk 2]
```diff
@@ -48,17 +48,32 @@
...
-                "Lung",
-                "Kidney",
-                "Gut",
+                "Brain",
+                "Breast",
+                "Development",
                 "Eye",
-                "Brain"
-            ],
+                "Eye - Other",
```
Comment (NoopDog) [inline json_schema/module/project/hca_bionetwork.json:57]: You can remove "Eye - Other"; this will be refined as the atlas becomes more focused.

**Review by NoopDog — CHANGES_REQUESTED** [review]
We can remove Eye-Other for now

File: json_schema/module/project/hca_bionetwork.json (hunk 2 above)
Comment (arschat) [reply json_schema/module/project/hca_bionetwork.json]: Thank you, Dave! Done.

**Review by amnonkhen — APPROVED** [review]

**Review by hannes-ucsc — APPROVED** [review]

**Review by NoopDog — APPROVED** [review]

**Pull request merged**